plugin/pkg/scheduler/factory/factory.go

```diff
@@ -531,12 +531,21 @@ func (factory *ConfigFactory) makeDefaultErrorFunc(backoff *podBackoff, podQueue
 			}
 			// Get the pod again; it may have changed/been scheduled already.
 			pod = &api.Pod{}
-			err := factory.Client.Get().Namespace(podID.Namespace).Resource("pods").Name(podID.Name).Do().Into(pod)
-			if err != nil {
-				if !errors.IsNotFound(err) {
-					glog.Errorf("Error getting pod %v for retry: %v; abandoning", podID, err)
+			backoff := time.Duration(100*time.Millisecond)
+			for {
```

Comment:
Yes - it may be - otherwise we will loose the pod and it will never be scheduled. This is on purpose.